qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/create/offer/OfferOverviewView.groovy

```diff
@@ -74,11 +78,14 @@ class OfferOverviewView extends VerticalLayout{
      */
     private static void generateProductGrid(Grid<ProductItemViewModel> grid) {
         try {
+            DecimalFormat euroFormat = new DecimalFormat("€#,##0.00")
```

Comment:
Two suggestions:

- Introduce a static class with a public static final field for the format, such that we can re-use it in other views when we need to display the currency

- Introduce a locale setting, that will load a currency based on the users locale setting. The default can be in Euro.